packages/@ember/-internals/glimmer/lib/utils/debug-stack.ts

```diff
@@ -16,6 +16,13 @@ if (DEBUG) {
   DebugStack = class DebugStack {
     private _stack: TemplateElement[] = [];
 
+    toString() {
```

Comment:
Lets add this to the new render stack, I suspect that the DebugStack will actually go away (in favor of using the same plumbing provided by the render stack).